lib/Sso/Service.php

```diff
@@ -104,6 +106,19 @@ public function getLogoutUrl() {
   }
 
   /**
+
+   * Return where the app should redirect after logging user out
+   *
+   * @return string url
+   */
+  public function getLogoutUrlWithUserUid($userUid) {
```

Comment:
I see that method overloading doe snot exist in php...
http://stackoverflow.com/questions/4697705/php-function-overloading